src/JestExt.ts

```diff
@@ -301,6 +301,15 @@ export class JestExt {
     this.triggerUpdateActiveEditor(editor)
   }
 
+  onDidSaveTextDocument(_editor: vscode.TextDocument) {
+    if (!this.jestProcessManager.numberOfProcesses || this.jestProcessManager.numberOfProcesses < 1) {
+      return
+    }
+
+    // each file save will/should trigger tests re-running
+    this.status.running('Running tests')

```

Comment:
the problem of doing it here is when users updated a document that didn't trigger jest run (such as a Readme.md), the status will be spinning forever... it's kind of hard to guess which file will trigger jest to re-run the tests since jest configuration could include custom matching logic/patterns...

Reply:
I didn't think about that issue. I will try to find some reasonable solution, thanks for the comment.